code/game/machinery/hologram.dm

```diff
@@ -56,6 +56,7 @@ var/global/const/HOLOPAD_MODE = RANGE_BASED
 	var/base_icon = "holopad-B"
 
 	var/allow_ai = TRUE
+	var/static/list/reachable_overmaps = list(OVERMAP_ID_SPACE)
```

Comment:
maybe should just grab whatever overmap is our location is on?
also might need third type for super long range for planet-space comms

Reply:
Thing is, a single z-level can be in multiple overmaps (exoplanet has an exoplanet loc and presumably a loc on its own overmap) so how do you tell which one to use?

Reply:
This is no longer relevant since I gave up on multiple overmap entities per site.

Reply:
I'm going to stick with the list version just so it's predictable what overmap the holopad reaches.